### PR TITLE
newgrp, pkgadd, tcpkill, xfreerdp: add Spanish translation

### PR DESCRIPTION
**Pull request by quique**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/linux/newgrp.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# newgrp
+
+> Cambia el grupo primario de pertenencia.
+
+- Cambia el grupo primario de pertenencia del usuario:
+
+`newgrp {{nombre_grupo}}`
+
+- Restablece el grupo primario de pertenencia al grupo del usuario predefinido en `/etc/passwd`:
```
Comment (navarroaxel) [inline pages.es/linux/newgrp.md:9]:
```suggestion
- Restablece el grupo primario de pertenencia al grupo por defecto del usuario `/etc/passwd`:
```

File: pages.es/linux/tcpkill.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# tcpkill
+
+> Mata la conexiones TCP en curso indicadas.
```
Comment (navarroaxel) [inline pages.es/linux/tcpkill.md:3]:
```suggestion
> Mata la conexiones TCP en curso especificadas.
```

File: pages.es/linux/xfreerdp.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# xfreerdp
+
+> Implementación libre del protocolo de escritorio remoto (RDP).
```
Comment (navarroaxel) [inline pages.es/linux/xfreerdp.md:3]:
```suggestion
> Implementación libre del protocolo de escritorio remoto (_Remote Desktop Protocol_).
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.es/linux/tcpkill.md (hunk 2 above)
Comment (navarroaxel) [reply pages.es/linux/tcpkill.md:3]: @quique what about the other suggestions?

Comment (quique) [reply pages.es/linux/tcpkill.md:3]: "indicadas" sounds more natural to me.
Does "especificadas" sound better in LatAm?

File: pages.es/linux/newgrp.md (hunk 1 above)
Comment (quique) [reply pages.es/linux/newgrp.md:9]: What do you think about "por omisión"?

File: pages.es/linux/tcpkill.md (hunk 2 above)
Comment (navarroaxel) [reply pages.es/linux/tcpkill.md:3]: for me, yes. Also, we use `especificado` in the translations.

File: pages.es/linux/newgrp.md (hunk 1 above)
Comment (navarroaxel) [reply pages.es/linux/newgrp.md:9]: I think `default` -> `por defecto`, it's a standard translation <https://apagayenciende.net/que-significa-por-defecto/>.

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]

**Pull request merged**